osmomath/sigfig_round.go

```diff
@@ -6,6 +6,9 @@ var pointOne = sdk.OneDec().QuoInt64(10)
 
 // SigFigRound rounds to a specified significant figure.
 func SigFigRound(d sdk.Dec, tenToSigFig sdk.Int) sdk.Dec {
+	if d.IsZero() {
+		return d
```

Comment:
We are completely missing tests for this function. We should add them

Reply:
Agreed, should be done in a separate PR